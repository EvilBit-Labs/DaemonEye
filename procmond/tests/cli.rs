use assert_cmd::prelude::*;
use predicates::prelude::*;
use std::process::Command;

#[test]
fn prints_expected_greeting() -> Result<(), Box<dyn std::error::Error>> {
    let mut cmd = Command::cargo_bin("procmond")?;
    cmd.assert().success().stdout(predicate::str::contains(
        "Hello from sentinel-lib to procmond!",
    ));
    Ok(())
}
