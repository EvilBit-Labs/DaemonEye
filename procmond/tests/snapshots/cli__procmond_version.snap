---
source: procmond/tests/cli.rs
expression: stdout
---
procmond 0.1.0
